### DISCUSSION TITLE
Notes as an overlay instead of separate Mode?

### DISCUSSION THREAD

**Discussion by mnalis** [Zverik/every_door#461] [Ideas]
Notes, as currently implemented in `2.0` (in https://github.com/Zverik/every_door/issues/47) leave something to be desired. While I understand they are to be a part of "[Walking papers mode](https://github.com/Zverik/every_door/issues/197)", there are several problems with using them effectively, e.g.:

- https://github.com/Zverik/every_door/issues/460
- https://github.com/Zverik/every_door/issues/459
- https://github.com/Zverik/every_door/issues/230

Which makes me wonder: would it be more sensible to make Notes a toggle-able overlay in other modes, instead of being a separate mode in itself? 

e.g. when in `Micromapping`, `Amenity` or `Entrance` modes, clicking a `Show/Hide notes button` would display or hide Notes in that same mode? (there should be a toggle instead of displaying Notes permanently, or otherwise Notes would often hide amenities below them)

**Comment by bfontaine** [comment]
+1 for this, because when you are close to a note you don’t see it unless you go in the dedicated mode. I could have checked and fixed several notes if I had seen them in POI mode.
